### PR TITLE
Fix language tags

### PR DESCRIPTION
**Pull request by ocean**
Some progress on #127 

Not sure how to deal with all "unusual" characters in language names to make them URLs. At least, it's ok to use Hugo's `safeURL` function to get rid of unsafe characters, but because the language pages and the links to them are generated in two different places, they need to have the same logic applied to them to produce the same URLs.

### CODE REVIEW

**Comment by ocean** [comment]
PR test is failing because the `'github.com/jina-ai/jina/'` is mentioned twice in `repositories.toml` 😢

**Comment by jai-deepsource** [comment]
@ocean Remove the duplicate in this PR?

**Comment by ocean** [comment]
Sorry, I left my test deployment files in, just removed them now. Ready for review, or at least further testing.

**Comment by ocean** [comment]
Argh, still doesn't handle languages with spaces in them. Needs more work, but I hope I've moved things forward a little bit.

**Review by rahul-deepsource — CHANGES_REQUESTED** [review]
@ocean: Thanks for the PR! The changes look good. I've left a few comments. Also, it'd be great if you could add a few test cases to make sure that this works: Add some languages with weird characters in them (`C#`, etc.) and test that the regex substitution works fine.

File: gfi/populate.py [hunk 1]
```diff
@@ -149,7 +149,10 @@ def get_repository_info(owner, name):
                 )
                 if repo_details:
                     REPOSITORIES.append(repo_details)
-                    TAGS[repo_details["language"]] += 1
+                    if repo_details["language"]:
+                        TAGS[repo_details["language"]] += 1
+                    else:
+                        TAGS["Unknown"] += 1
```
[the remark this answers is not in the document]
Comment (ocean) [reply gfi/populate.py:155]: Ok, good idea about leaving out repos with unknown languages.

I will add some test cases too 👍

**Comment by sanket-deepsource** [comment]
This has been handled in a recent release.

**Pull request closed without merging**